rules/linux/discovery_kernel_module_enumeration.toml

```diff
@@ -79,7 +86,7 @@ reference = "https://attack.mitre.org/tactics/TA0007/"
 
 [rule.new_terms]
 field = "new_terms_fields"
-value = ["host.id", "process.command_line", "process.parent.executable"]
+value = ["process.parent.command_line", "process.command_line", "process.parent.executable"]
```

Comment:
Do we want this to exclude the whole environment or keep it for a specific host?

Reply:
Looking to exclude the whole environment, given its unlikeliness to trigger on these conditions.